common/buildings/wc_common_buildings.txt

```diff
@@ -0,0 +1,760 @@
+﻿### aviary complex
+
+aviary_complex_01 = {
+	construction_time = quick_construction_time
+
+	can_construct_potential = {
+		building_aviary_complex_requirement_terrain = yes
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		culture = {
+			OR = {
+				scope:holder.culture = {
+					OR = {
+						has_innovation = innovation_taming_the_skies
+						has_innovation = innovation_flying_beasts
+					}
+				}	
+				has_special_aerial_maa_trigger = yes
+				has_special_massive_aerial_maa_trigger = yes
```

Comment:
About these triggers.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/187270874-4969bc63-c3b6-41aa-9e33-ea911d525d9c.png)
![image](https://user-images.githubusercontent.com/46384992/187270848-53214747-9ce4-4816-a568-40e5697c0b9a.png)

</details>

Basically, it's better to create scripted triggers that united these 2 conditions: something like `can_unlock_innovation_dragon`, etc. for other triggers.

Reply:
This is the only tech of massive aerial that has 2 cultures unlocking it. I can make it for the dragon (can_unlock_innovation_dragon), do I need to do it for the rest? It will just be 1 culture per trigger?

Reply:
> This is the only tech of massive aerial that has 2 cultures unlocking it. I can make it for the dragon (can_unlock_innovation_dragon), do I need to do it for the rest?

I would do the same for other innovations, yes. It's done so you don't need to change both innovation `potential` and `has_special_maa_trigger` when it's needed.

> It will just be 1 culture per trigger?

Sometime yeah.